app/models/book.rb

```diff
@@ -0,0 +1,3 @@
+class Book < ApplicationRecord
+  has_many :reviews
```

Comment:
We can also specify belong to dependency here as well
ref: https://guides.rubyonrails.org/association_basics.html#options-for-belongs-to-dependent

```suggestion
  has_many :reviews, dependent: :destroy
```